app/src/rgb_underglow.c

```diff
@@ -187,20 +215,35 @@ static int zmk_rgb_underglow_init(struct device *_arg)
 	}
 
     state = (struct rgb_underglow_state){
-        hue: 0,
-        saturation: 100,
-        brightness: 100,
-        animation_speed: 3,
-        current_effect: 0,
+        hue: CONFIG_ZMK_RGB_UNDERGLOW_HUE_START,
+        saturation: CONFIG_ZMK_RGB_UNDERGLOW_SAT_START,
+        brightness: CONFIG_ZMK_RGB_UNDERGLOW_BRT_START,
+        animation_speed: CONFIG_ZMK_RGB_UNDERGLOW_SPD_START,
+        current_effect: CONFIG_ZMK_RGB_UNDERGLOW_EFF_START,
         animation_step: 0,
+#ifdef CONFIG_ZMK_RGB_UNDERGLOW_ON_START
         on: true
+#else
+        on: false
+#endif
```

Comment:
Is there a better way to do this? It feels pretty verbose just to check if something is defined.